users/vieting/experiments/switchboard/ctc/feat/experiments.py

```diff
@@ -482,140 +758,207 @@ def run_scf_specaug_sort():
+    returnn_args = {
+        "batch_size": 5000,
+        "rasr_binary_path": RASR_BINARY_PATH,
+        "rasr_loss_corpus_path": rasr_loss_corpus_path,
+        "rasr_loss_corpus_segments": rasr_loss_corpus_segments,
+        "rasr_loss_lexicon_path": rasr_loss_lexicon_path,
+        "datasets": returnn_datasets,
+        "conformer_type": "wei",
+        "audio_perturbation": True,
+        "use_multi_proc_dataset": True,
+        "specaug_old": {"max_feature": 8},
+        "extra_args": {
+            "audio_perturb_runner": CodeWrapper("WaveformPerturbation(**audio_perturb_args)"),
+            "conv_pad_seq_len_to_power": 1.5,
+            "accum_grad_multiple_step": 2,
+        },
+    }
+    # usually the lr args would need to be adapted to fit with the checkpoint,
+    # but experiments showed that restarting the scheduling is better
+    lr_args = {
+        "peak_lr": 4e-4,
+        "start_lr": 1.325e-05,
+        "end_lr": 1e-5,
+        "increase_epochs": 180,
+        "decrease_epochs": 180,
+        "final_epochs": 0,
+    }
+
+    perturbation_args = [
+        {"codecs": [{"encoding": "ULAW", "prob": 0.3, "minimum": 250, "maximum": 260}]},
+        {"codecs": [{"encoding": "ULAW", "prob": 0.7, "minimum": 250, "maximum": 260}]},
+        {"codecs": [{"encoding": "ULAW", "prob": 0.7, "minimum": 240, "maximum": 270}]},
+        {"codecs": [{"encoding": "ULAW", "prob": 1, "minimum": 250, "maximum": 260}]},
+        {"tempo": {"prob": 0.3, "minimum": 0.83, "maximum": 1.17}},
+        {"tempo": {"prob": 0.8, "minimum": 0.83, "maximum": 1.17}},
+        {"tempo": {"prob": 0.3, "minimum": 0.7, "maximum": 1.3}},
+        {"tempo": {"prob": 0.8, "minimum": 0.7, "maximum": 1.3}},
+        {"tempo": {"prob": 1, "minimum": 0.83, "maximum": 1.17}},
+        {"tempo": {"prob": 1, "minimum": 0.7, "maximum": 1.3}},
+        {"speed": {"prob": 0.5, "minimum": 0.88, "maximum": 1.12}},
+        {"speed": {"prob": 0.3, "minimum": 0.88, "maximum": 1.12}},
+        {"speed": {"prob": 0.7, "minimum": 0.88, "maximum": 1.12}},
+        {"speed": {"prob": 0.6, "minimum": 0.8, "maximum": 1.2}},
+        {"speed": {"prob": 0.6, "minimum": 0.9, "maximum": 1.1}},
+        {"speed": {"prob": 1, "minimum": 0.9, "maximum": 1.1}},
+        {"speed": {"prob": 1, "minimum": 0.8, "maximum": 1.2}},
+        {"non_linearity": {"prob": 0.7, "minimum": 0.9, "maximum": 1.1}},
+        {"non_linearity": {"prob": 0.3, "minimum": 0.9, "maximum": 1.1}},
+        {"non_linearity": {"prob": 1, "minimum": 0.9, "maximum": 1.1}},
+        {"preemphasis": {"prob": 0.3, "minimum": 0.94, "maximum": 1.0, "default": 0.97}},
+        {"preemphasis": {"prob": 0.7, "minimum": 0.94, "maximum": 1.0, "default": 0.97}},
+        {"preemphasis": {"prob": 1.0, "minimum": 0.94, "maximum": 1.0, "default": 0.97}},
+        {"preemphasis": {"prob": 0.7, "minimum": 0.90, "maximum": 1.0, "default": 0.97}},
+        {"preemphasis": {"prob": 0.7, "minimum": 0.90, "maximum": 1.0, "default": 0.95}},
```

Comment:
You have preemphasis experiments here but always initialize with the checkpoint which did not use preemphasis.